core/auth/creation_test.go

```diff
@@ -0,0 +1,40 @@
+package auth
+
+import (
+	"testing"
+
+	"fmt"
+
+	"github.com/bohdanlisovskyi/telegrambot/tbot"
+)
+
+//var mockadapter = adapter.Message{
```

Comment:
If you don't need this commented code - delete it. But, I think you gonna use it. Am I right?

Reply:
I need ask how to mock telegram message correctly, than can delete this commented code